src/store/actions.js

```diff
@@ -0,0 +1,8 @@
+export const addTask = (title) => ({
+  type: "ADD_TASK",
+  payload: {
+    id: Date.now(),
+    title,
+    createdDate: new Date(),
```

Comment:
you can go through those website: https://eslint.org/docs/latest/rules/object-shorthand
```suggestion
    id: Date.now(),
    title: title,
    createdDate: new Date(),
```